scripts/update_packages.ts

```diff
@@ -0,0 +1,162 @@
+// import yargs from 'yargs/yargs'
+// import { hideBin } from 'yargs/helpers'
+import fs from 'fs'
+
+const packages = {
```

Comment:
Handy routine. Did you populate with some scanning script, or was this by hand?

Reply:
By hand. Basically looking at what was in helltool, sdk-codegen and npm. Had some free time Monday afternoon and was a little bit irritated after issues with my first fix on Friday evening